### PR TITLE
Generate scale.py file

### PR DESCRIPTION
**Pull request by subash-suresh**
**What does this Pull Request accomplish?**

Generates the `scale.py` files using the scrapigen metadata.

The following changes have been made in the PR,
- Added a mako template called `scale.py.mako` for `scale.py` file generation.
- Updated the `ATTRIBUTE_CHANGE_SET` dictionary to rectify some name changes in the properties generated.
- Updated the `PYTHON_CLASS_ENUM_MERGE_SET` to import additional enums.
- Updated the `script_info` file to include details on the scale.py generation.

**Why should this Pull Request be merged?**
Implemented [Task 2339424](https://ni.visualstudio.com/DevCentral/_workitems/edit/2339424): Generate the scale.py

### CODE REVIEW

**Review by keer1111 — APPROVED** [review]

**Review by zhindes — COMMENTED** [review]
Just to confirm, no changes to generated scale.py?

File: src/codegen/metadata/script_info.py [hunk 1]
```diff
@@ -32,5 +32,9 @@
             "relativeOutputPath": "_task_modules\\channels\\do_channel.py",
```
[the remark this answers is not in the document]
Comment (subash-suresh) [reply src/codegen/metadata/script_info.py]: Yes, there is no changes in the generated file

**Comment by subash-suresh** [comment]
> Just to confirm, no changes to generated scale.py?

Yes, that's right

**Review by zhindes — APPROVED** [review]

**Review by bkeryan — COMMENTED** [review]
Looks good to me.

**Pull request merged**